test/modules/TestAnalyzer/Settings.cs

```diff
@@ -39,12 +40,7 @@ class Settings
             this.WebhostPort = Preconditions.CheckNonWhiteSpace(webhostPort, nameof(webhostPort));
             this.ToleranceInMilliseconds = Preconditions.CheckRange(tolerance, 0);
             this.StoragePath = storagePath;
-            this.TestInfo = testInfo.Split(",", StringSplitOptions.RemoveEmptyEntries)
-                                .Select(x => (KeyAndValue: x, SplitIndex: x.IndexOf('=')))
-                                .Where(x => x.SplitIndex >= 1)
-                                .ToDictionary(
-                                    x => x.KeyAndValue.Substring(0, x.SplitIndex),
-                                    x => x.KeyAndValue.Substring(x.SplitIndex + 1, x.KeyAndValue.Length - x.SplitIndex - 1));
+            this.TestInfo = ModuleUtil.ParseTestInfo(testInfo);
```

Comment:
do you need to add Device id in TestInfo?

Reply:
I don't think I needed so I didn't add. Lmk if you disagree.